components/carbon-authenticators/thrift-authenticator/org.wso2.carbon.identity.authenticator.thrift/src/main/java/org/wso2/carbon/identity/thrift/authentication/internal/persistance/ThriftAuthenticationJDBCPersistenceManager.java

```diff
@@ -103,17 +103,24 @@ private void initDataSource() throws AuthenticationException {
         }
     }
 
+    @Deprecated
```

Comment:
Need to put a comment why this is deprecated and the alternative to be used.

Reply:
Added